lib/validations.js

```diff
@@ -383,6 +384,18 @@ function validateUniqueness(attr, conf, err, options, done) {
   }.bind(this));
 }
 
+/*!
+ * Date validator
+ */
+function validateDate(attr, conf, err) {
```

Comment:
It turns out that `conf` is needed, otherwise the tests failed.
It's a required attribute for similar functions defined inside `validators` (in line397), even though it's not being used.